### PR TITLE
backports/pysdl2: explicitly name subpackages

### PR DESCRIPTION
**Pull request by f-fl0**

### CODE REVIEW

File: backports/py-pysdl2/APKBUILD [hunk 1]
```diff
@@ -3,15 +3,15 @@
 pkgname=py-pysdl2
 _pkgname=py-sdl2
```
Comment (at-wat) [inline backports/py-pysdl2/APKBUILD:4]:
```suggestion
_pkgname=pysdl2
```

File: backports/py-pysdl2/APKBUILD [hunk 2]
```diff
@@ -3,15 +3,15 @@
...
+pkgrel=6
 pkgdesc="Python ctypes wrapper around SDL2"
 url="https://github.com/py-sdl/py-sdl2"
 arch="noarch"
 license="CC0 Public Domain Dedication"
 depends="sdl2 py-pysdl2-dll"
 makedepends="python2-dev python3-dev py-setuptools"
 options="!check" #no testsuite
-subpackages="$pkgname-doc py2-$_pkgname:_py2 py3-$_pkgname:_py3"
+subpackages="$pkgname-doc py2-${_pkgname//-/}:_py2 py3-${_pkgname//-/}:_py3"
```
Comment (at-wat) [inline backports/py-pysdl2/APKBUILD:14]:
```suggestion
subpackages="$pkgname-doc py2-$_pkgname:_py2 py3-$_pkgname:_py3"
```

File: backports/py-pysdl2/APKBUILD [hunk 3]
```diff
@@ -3,15 +3,15 @@
...
 source="$_pkgname-$pkgver.tar.gz::https://github.com/py-sdl/$_pkgname/archive/$pkgver.tar.gz"
```
Comment (at-wat) [inline backports/py-pysdl2/APKBUILD:15]:
```suggestion
source="$_pkgname-$pkgver.tar.gz::https://github.com/py-sdl/py-sdl2/archive/$pkgver.tar.gz"
```

**Review by at-wat — COMMENTED** [review]
not sure the suggestion works

**Comment by f-fl0** [comment]
The builds passed this time but some variables names needed to be adjusted. Maybe there is a cleaner way. @at-wat what do you think?

**Review by at-wat — APPROVED** [review]
LGTM

**Pull request merged**